tools/scuffed-scraper/main.py

```diff
@@ -0,0 +1,612 @@
+# DTU Scuffed Course Scraper
+# Copyright (C) 2023 Sofus Albert Høgsbro Rose
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import sys
+
+SUPPORTED_PY_MAJOR = 3
+SUPPORTED_PY3_VERSIONS = [11, 12, 13, 14]
+if not all(
+	[
```

Comment:
you are using 8 spaces for indentation, could we reduce to 4? This will be consistent with the other codes in the repo

Reply:
My GitHub settings are set to render tabs with 4. I'd be happy to run `black` instead of `tan` if you'd prefer, but I'd rather do that after the rest of the review (to keep nice diffs).